create_virtualmachine_with_subnet_in_different_resource_group.yml

```diff
@@ -1,7 +1,13 @@
 # Description
 # ===========
 # This playbook create an Azure VM with public IP, and subnet in another resource group.
-
+# This Script will create two resource group, defined by variables resource_group and resource_group_secondary. Then it creates VM in resource_group and virtual network with defined subnets,nsg rules in resource_group_secondary.
```

Comment:
"This Script will create two resource groups,"

Reply:
Please break this paragraph into two lines to avoid a long line

Reply:
Sure let me add this to readme instead.